oxidation/crates/parsec_core_fs/migrations/2022-04-26-103106_chunks/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+-- This file should undo anything in `up.sql`
+DROP TABLE IF EXISTS chunks;
```

Comment:
You must create tests if you add a way to downgrade the datamodel.

I think it would be better to not provide any way to downgrade given it's not the goal of this PR (and YAGNI), and a poorly tested feature we might be temped to use in the future is far worst than no feature at all

Reply:
I use the migrations to generate the schema.rs file, that's how diesel work